nin/dasBoot/loop.js

```diff
@@ -18,10 +18,10 @@ function createLoop(options) {
       that.deltaTime += that.time - that.oldTime;
       that.oldTime = that.time;
       while (that.deltaTime >= frameLength) {
+        that.deltaTime -= frameLength;
```

Comment:
If a layer's update method crashes before deltaTime gets adjusted, the demo will try to "catch up on time" by calling update tons of times in a row after the crash gets fixed. This will crash the execution environment if you are unlucky. If deltaTime gets adjusted beforehand, on the other hand, the loop will not think the the demo is behind on schedule and does not attempt to "catch up".